cpg-core/src/test/kotlin/de/fraunhofer/aisec/cpg/frontends/cpp/CXXLanguageFrontendTest.kt

```diff
@@ -1413,6 +1413,42 @@ internal class CXXLanguageFrontendTest : BaseTest() {
         assertRefersTo(callee.rhs, singleParam)
     }
 
+    @Test
```

Comment:
We should test if the resolution works with and without the `ControlFlowSensitiveDFGPass`

Reply:
Is that really needed? I thought the idea was to merge two DFG passes anyway?

Reply:
Yes, but this means that the `FunctionPointerCallResolver` has to handle the case of the state after the control flow sensitive pass and the one without it. It's just to make sure that we don't miss something because our tests always have the "control flow sensitive" configuration by accident or so (just as we missed that the old version didn't work properly with a slightly different but acceptable configuration)

Reply:
Ok, I duplicated the test with and without the control-flow-DFG.